CHANGELOG.md

```diff
@@ -6,11 +6,13 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic
 Versioning](https://semver.org/spec/v2.0.0.html).
 
+
```

Comment:
We should add `CHANGELOG.md` to our markdownlint check (https://github.com/itk-dev/os2loop/blob/develop/package.json#L10), e.g. 

```
markdownlint *.md 'docs/**/*.md' 'web/profiles/custom/os2loop/**/*.md' --ignore 'web/profiles/custom/os2loop/**/node_modules/**/*.md'
```

Reply:
Added